src/clj/ataru/kk_application_payment/kk_application_payment.clj

```diff
@@ -0,0 +1,70 @@
+(ns ataru.kk-application-payment.kk-application-payment
+  "Logic related to kk application processing payments, AKA hakemusmaksu. The basic spec is that
+   non-exempt non-EU native applicants should be charged an application fee once per semester.
+   NB! Semester is defined here by the start date of the actual first higher education semester,
+   not the application date."
+  (:require [ataru.kk-application-payment.kk-application-payment-store :as store]
+            [taoensso.timbre :as log]))
+
+; TODO: application payments are only charged for hakus starting after 1.1.2025 AND targeting kausi_s 2025
```

Comment:
Onko tämä TODO tarkoitus tehdä milloin? Vai onko se jo tehty ja tämä vaan unohtunut? Alempana start-term-valid?

Reply:
Hyvä huomio: tuo jälkimmäinen osio on toteutettu jo, mutta haun alkamispäivän rajausta ei vielä ole. Olin ajatuksissani jättänyt sen OK-556:n yhteydessä toteutettavaksi - jatkoin työtä saumattomasti siitä eteenpäin, ja tämä olisi tarkoitus aluksi mergetä featurebranchiin OK-485 ennen masteria.

Reply:
Voisi varmuudeksi merkitä todo-kommenttiin tiketin missä se tehdään